app/models/notifier.rb

```diff
@@ -161,19 +173,44 @@ def changeset_comment_notification(comment, recipient)
       @changeset_comment = comment.changeset.tags["comment"].presence
       @time = comment.created_at
       @changeset_author = comment.changeset.user.display_name
+      @user_message_author = @commenter
 
       subject = if @owner
                   I18n.t("notifier.changeset_comment_notification.commented.subject_own", :commenter => @commenter)
                 else
                   I18n.t("notifier.changeset_comment_notification.commented.subject_other", :commenter => @commenter)
                 end
 
+      attach_user_avatar(comment.author)
+
       mail :to => recipient.email, :subject => subject
     end
   end
 
   private
 
+  def set_shared_template_vars
+    @root_url = root_url(:host => SERVER_URL)
+    attach_project_logo
```

Comment:
Good point, I've made `attach_project_logo` a `before_action` filter. `set_shared_template_vars` is still there and still sets `@root_url`